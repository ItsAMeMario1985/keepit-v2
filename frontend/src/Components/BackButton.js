import { ReactComponent as BackButtonSvg } from '../Assets/back.svg'
import { useHistory, useLocation } from 'react-router-dom'

export default function BackButton({ onClick, buttonText }) {
  function handleOnClick() {
    history.goBack()
  }
  const history = useHistory()
  return <BackButtonSvg onClick={handleOnClick}></BackButtonSvg>
}
