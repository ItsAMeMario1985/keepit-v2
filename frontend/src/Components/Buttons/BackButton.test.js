import { render } from '@testing-library/react'
import BackButton from './BackButton'
describe('BackButton', () => {
  it('renders correctly', () => {
    const { container } = render(<BackButton />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
