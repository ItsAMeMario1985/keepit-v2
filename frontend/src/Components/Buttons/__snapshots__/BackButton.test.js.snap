// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton renders correctly 1`] = `
<svg>
  back.svg
</svg>
`;
